metrics/sinks/stackdriver/stackdriver.go

```diff
@@ -42,6 +42,9 @@ const (
 	maxTimeseriesPerRequest = 200
 	// 2 seconds on SD side, 1 extra for networking overhead
 	sdRequestLatencySec = 3
+
+	// Upon backend issues, buffer up to 5 batches.
+	batchesBufferSize = 5
```

Comment:
Does this change impact memory usage?

Reply:
It puts a limit on the memory usage: existing code would always spin up a new goroutine, so there was no bound on the memory.